yaksh/templates/yaksh/course_detail.html

```diff
@@ -6,16 +6,32 @@
 
 {% block script %}
 <script language="JavaScript" type="text/javascript" src="{{ URL_ROOT }}/static/yaksh/js/course.js"></script>
+<script type="text/javascript" src="https://cdnjs.cloudflare.com/ajax/libs/froala-editor/2.5.1/js/froala_editor.min.js"></script>
+<script src="https://code.jquery.com/ui/1.9.1/jquery-ui.js"></script>
```

Comment:
jquery imported in base.html is jquery min. This jquery is jquery ui which is required for js editor